ros2controlcli/test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
We are always dragging this old copyrights.... @bmagyar is this correct or should we update them?

Reply:
that's ok as long as it's a full copy of the file.

Reply:
it's old boilerplate